src/wazuh_modules/wmodules.c

```diff
@@ -299,7 +299,7 @@ int modulesSync(char* args) {
     for (cur_module = wmodules; cur_module; cur_module = cur_module->next) {
         if (strstr(args, cur_module->context->name)) {
```

Comment:
I suggest you improve the modules filter here by doing a `strncmp`.

Reply:
This will prevent an unexpected match. Fixed!